bin/pyAFQ_dki

```diff
@@ -28,7 +28,9 @@ parser.add_option("-o", "--out_dir",
                   help="full path to directory for files to be saved (will be created if it doesn't exist)")
 
 parser.add_option("-m", "--mask", help="mask file")
+parser.add_option('-b', '--b0_threshold', help="b0 threshold", default=0)
+
 opts, args = parser.parse_args()
 
 dki.fit_dki(opts.dwi, opts.bval, opts.bvec, mask=opts.mask,
-            out_dir=opts.out_dir)
+            out_dir=opts.out_dir, b0_threshold=float(opts.b0_threshold))
```

Comment:
Just a (hopefully helpful) note: you can specify the type right on the option. Should give the user a bit more info, fail at a better point for bad inputs, etc.